sklearn/ensemble/_bagging.py

```diff
@@ -265,6 +272,21 @@ def __init__(
         self.random_state = random_state
         self.verbose = verbose
 
+    def _validate_estimator(self, default=None):
```

Comment:
If we have the deprecation in the base class, I assume that we will not need this part of the code.